modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
+	@priority = 5
+	test AssertCRUDWithObjectDefinition {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object definition created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 193625",
+				objectName = "CustomObject193625",
+				pluralLabelName = "Custom Objects 193625");
+		}
+
+		task ("and Given a custom object definition updated") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 193625");
+
+			ObjectAdmin.editCustomObject(fieldLabelName = "New object definition updated");
+		}
+
+		task ("and Given a custom object definition deleted") {
+			ObjectAdmin.deleteObjectViaAPI(objectName = "CustomObject193625");
+		}
+
+		task ("When accessing the audit admin") {
+			Audit.openAuditAdmin();
+		}
+
+		task ("Then assert that the audit can see the custom object definition created, updated and deleted") {
```

Comment:
Combine this into one task:

`Then the Add, Update, and Delete events appear in the Audit records`